pkg/routes/ui.go

```diff
@@ -22,7 +22,7 @@ import (
 	"k8s.io/apiserver/pkg/server/mux"
 )
 
-const dashboardPath = "/api/v1/namespaces/kube-system/services/kubernetes-dashboard/proxy"
+const dashboardPath = "/api/v1/namespaces/kube-system/services/https:kubernetes-dashboard:/proxy/"
```

Comment:
there's no guarantee that the dashboard was deployed using the addon manifest... see discussion in https://github.com/kubernetes/kubernetes/pull/53766 and https://github.com/kubernetes/kubernetes/pull/56074#issuecomment-345774687 about why this should be removed, not perpetuated.

Hardcoding a different value here does not make it more wrong, but also does not make it more correct. I still intend to include deprecation notes in 1.9 for the /ui redirect, sweep and remove references to it, and remove it in a future version.

Reply:
I can understand it. Can we update it just before to make it work or should it be kept broken?

> there's no guarantee that the dashboard was deployed using the addon manifest...

I agree. We try to enforce HTTPS setup everywhere as recommended one.

Reply:
+1 to removing this, but I don't think it should block this PR. Can we release note that it will be going away in 1.10 and make this change for 1.9?

Reply:
> Can we release note that it will be going away in 1.10 and make this change for 1.9?

That seems ok. Something like this?

> Updated Dashboard add-on to version 1.8.0.
> * The dashboard add-on now deploys with https enabled
> * The dashboard can be accessed via `kubectl proxy` at http://localhost:8001/api/v1/namespaces/kube-system/services/https:kube-dashboard:/proxy/
> * The `/ui` redirect is deprecated and will be removed in a future release.

Reply:
Done.